components/RecentEpisodes.vue

```diff
@@ -19,55 +21,82 @@ onMounted(async () => {
 </script>
  
 <template>
-  <div v-if="dataLoaded">
-    <div class="latest-episode grid justify-content-center">
+  <div>
+    <div v-if="dataLoaded" class="latest-episode grid justify-content-center">
       <div class="col-12 xl:col-8 p-0">
-        <img class="latest-episode-image" :src="episodes[0].attributes['image-main'].url" :alt="episodes[0].attributes['image-main']['alt-text']" />
+        <!-- <nuxt-link :to="`/episodes/${episodes[0].attributes.slug}`">
+          <img class="latest-episode-image" :src="episodes[0].attributes['image-main'].url" :alt="episodes[0].attributes['image-main']['alt-text']" />
+        </nuxt-link>-->
+        <v-image-with-caption
+          :image="episodes[0].attributes['image-main'].url"
```

Comment:
this doesn't have the .replace() I added to your branch yesterday. I tried making it all nice with the helper func: publisherImageFormatter 

but it wouldn't work... so I just in lined it like this:
:image="episodes[0].attributes['image-main'].template.replace('%s/%s/%s/%s', '%width%/%height%/c/%quality%')"

Reply:
this enables the smart srcset system we have